python/lsst/meas/base/noiseReplacer.py

```diff
@@ -382,7 +382,7 @@ class ImageNoiseGenerator(NoiseGenerator):
 
     def __init__(self, img):
         """
-        img: an afwImage.ImageF
```

Comment:
These docstrings should have a `!` if you're going to use doxygen.  Given the changes coming down the track, I don't feel the need to doxygenate here.